server/boot/root.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(server) {
```

Comment:
By remove, you mean delete from strong-mesh-models? I think it should be here, its useful when we run this repo stand-alone, its just that the REST API exported from strong-mesh-models shouldn't include it. If that's possible...